app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/SpeakerButton.kt

```diff
@@ -2,21 +2,35 @@ package com.wire.android.ui.calling.controlButtons
 
 import androidx.compose.foundation.Image
 import androidx.compose.foundation.layout.width
+import androidx.compose.material3.IconButton
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import coil.compose.rememberAsyncImagePainter
 import com.wire.android.R
 import com.wire.android.ui.theme.wireDimensions
-import com.wire.android.util.getUriFromDrawable
 
 @Composable
 fun SpeakerButton() {
-    Image(
```

Comment:
had to wrap it in a `IconButton` so the dimensions were the same as the other buttons